relay-event-normalization/src/normalize/span/country_subregion.rs

```diff
@@ -0,0 +1,261 @@
+        ("NL", Subregion::WesternEurope),
+        ("NZ", Subregion::AustraliaAndNewZealand),
+        ("NI", Subregion::CentralAmerica),
+        ("NE", Subregion::WesternAfrica),
+        ("NG", Subregion::WesternAfrica),
+        ("NO", Subregion::NorthernEurope),
+        ("OM", Subregion::WesternAsia),
+        ("PK", Subregion::SouthernAsia),
+        ("PW", Subregion::Micronesia),
+        ("PA", Subregion::CentralAmerica),
+        ("PG", Subregion::Melanesia),
+        ("PY", Subregion::SouthAmerica),
+        ("PE", Subregion::SouthAmerica),
+        ("PH", Subregion::SouthEasternAsia),
+        ("PL", Subregion::EasternEurope),
+        ("PT", Subregion::SouthernEurope),
+        ("QA", Subregion::WesternAsia),
+        ("RO", Subregion::EasternEurope),
+        ("RU", Subregion::EasternEurope),
+        ("RW", Subregion::EasternAfrica),
+        ("KN", Subregion::Caribbean),
+        ("LC", Subregion::Caribbean),
+        ("VC", Subregion::Caribbean),
+        ("WS", Subregion::Polynesia),
+        ("SM", Subregion::SouthernEurope),
+        ("ST", Subregion::MiddleAfrica),
+        ("SA", Subregion::WesternAsia),
+        ("SN", Subregion::WesternAfrica),
+        ("RS", Subregion::SouthernEurope),
+        ("SC", Subregion::EasternAfrica),
+        ("SL", Subregion::WesternAfrica),
+        ("SG", Subregion::SouthEasternAsia),
+        ("SK", Subregion::EasternEurope),
+        ("SI", Subregion::SouthernEurope),
+        ("SB", Subregion::Melanesia),
+        ("SO", Subregion::EasternAfrica),
+        ("ZA", Subregion::SouthernAfrica),
+        ("SS", Subregion::EasternAfrica),
+        ("ES", Subregion::SouthernEurope),
+        ("LK", Subregion::SouthernAsia),
+        ("SD", Subregion::NorthernAfrica),
+        ("SR", Subregion::SouthAmerica),
+        ("SZ", Subregion::SouthernAfrica),
+        ("SE", Subregion::NorthernEurope),
+        ("CH", Subregion::WesternEurope),
+        ("SY", Subregion::WesternAsia),
+        ("TJ", Subregion::CentralAsia),
+        ("TZ", Subregion::EasternAfrica),
+        ("TH", Subregion::SouthEasternAsia),
+        ("TL", Subregion::SouthEasternAsia),
+        ("TG", Subregion::WesternAfrica),
+        ("TO", Subregion::Polynesia),
+        ("TT", Subregion::Caribbean),
+        ("TN", Subregion::NorthernAfrica),
+        ("TR", Subregion::WesternAsia),
+        ("TM", Subregion::CentralAsia),
+        ("TV", Subregion::Polynesia),
+        ("UG", Subregion::EasternAfrica),
+        ("UA", Subregion::EasternEurope),
+        ("AE", Subregion::WesternAsia),
+        ("GB", Subregion::NorthernEurope),
+        ("US", Subregion::NorthernAmerica),
+        ("UY", Subregion::SouthAmerica),
+        ("UZ", Subregion::CentralAsia),
+        ("VU", Subregion::Melanesia),
+        ("VE", Subregion::SouthAmerica),
+        ("VN", Subregion::SouthEasternAsia),
+        ("YE", Subregion::WesternAsia),
+        ("ZM", Subregion::EasternAfrica),
+        ("ZW", Subregion::EasternAfrica),
```

Comment:
Is this the entire list, wikipedia says there are 249 countries, the list is 193.

Reply:
Thanks, I'm going to go through it and double check everything!